plugins/terraform/resource_load_balancer.go

```diff
@@ -112,7 +112,8 @@ func resourceLayer0LoadBalancerCreate(d *schema.ResourceData, meta interface{})
 	ports := expandPorts(d.Get("port").(*schema.Set).List())
 	healthCheck := expandHealthCheck(d.Get("health_check"))
 
-	if !healthCheck {
+	blankCheck := models.HealthCheck{}
```

Comment:
I don't think I'm catching the difference between this recommended
```
blankCheck := &models.HealthCheck{}
if healthCheck == *blankCheck {
```
and my own
```
blankCheck := models.HealthCheck{}
if healthCheck == blankCheck {
```